src/DataLocalizer.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace League\ISO3166;
+
+use DomainException;
+use Generator;
+use InvalidArgumentException;
+use Traversable;
+
+class DataLocalizer
```

Comment:
Not at all adding an interface would effectively make localization intl independent. I'll look into it